nodejs/week3/src/backend/api/reservations.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const pool = require("./../database");
+
+// Returns all reservations	
+router.get("/", (request, response) => {
+    pool.query("SELECT * FROM reservation", function(error, results) {
+      if (error) {
+        console.log(error);
+        response.send("No reservations were found, please check message error")
+      } else {
+      return response.json(results);
+      
+  } next();
+  }
+  ) 
+  });
+  
+  
+  // Adds a new reservations	
+  router.post("/", (request, response) => {
+    const { Number_of_guests, Meal_id, Created_date } = request.body;
+    const data = { 
+    Number_of_guests: Number_of_guests,
```

Comment:
Any reason for using this naming style?

Reply:
Modify in last commit 0c7728991bd1f785fb8a2affa641961574b92c4e. 
Thanks for your time!